test/acceptance/features/steps/app.py

```diff
@@ -51,6 +53,9 @@ def get_generation(self):
 @step(u'jsonpath "{json_path}" on "{res_name}" should return no value')
 def resource_jsonpath_value(context, json_path, res_name, json_value=""):
     openshift = Openshift()
+    json_path = Template(json_path).substitute(scenario_id=scenario_id(context))
```

Comment:
how about that we extract `Template(json_path).substitute(scenario_id=scenario_id(context))` into a common function and use it where needed, so that we can minimize DRY?

Reply:
Yep, that's probably for the better.  I'll go change that.